xml/mdns_device-info_txt.xml

```diff
@@ -0,0 +1,134 @@
+      <param pos="0" name="os.family" value="Mac OS X"/>
+      <param pos="0" name="os.product" value="Mac OS X"/>
+      <param pos="0" name="os.version" value="10.6"/>
+   </fingerprint>
+
+   <fingerprint pattern="^osxvers=9">
+      <description>Mac OS X 10.5 (Leopard)</description>
+      <example>osxvers=9</example>
+      <param pos="0" name="os.certainty" value="0.85"/>
+      <param pos="0" name="os.vendor" value="Apple"/>
+      <param pos="0" name="os.family" value="Mac OS X"/>
+      <param pos="0" name="os.product" value="Mac OS X"/>
+      <param pos="0" name="os.version" value="10.5"/>
+   </fingerprint>
+
+   <fingerprint pattern="^osxvers=8">
+      <description>Mac OS X 10.4 (Tiger)</description>
+      <example>osxvers=8</example>
+      <param pos="0" name="os.certainty" value="0.85"/>
+      <param pos="0" name="os.vendor" value="Apple"/>
+      <param pos="0" name="os.family" value="Mac OS X"/>
+      <param pos="0" name="os.product" value="Mac OS X"/>
+      <param pos="0" name="os.version" value="10.4"/>
+   </fingerprint>
+
+   <fingerprint pattern="^osxvers=7">
+      <description>Mac OS X 10.3 (Panther)</description>
+      <example>osxvers=7</example>
+      <param pos="0" name="os.certainty" value="0.85"/>
+      <param pos="0" name="os.vendor" value="Apple"/>
+      <param pos="0" name="os.family" value="Mac OS X"/>
+      <param pos="0" name="os.product" value="Mac OS X"/>
+      <param pos="0" name="os.version" value="10.3"/>
+   </fingerprint>
+
+   <fingerprint pattern="^osxvers=6">
+      <description>Mac OS X 10.2 (Jaguar)</description>
+      <example>osxvers=6</example>
+      <param pos="0" name="os.certainty" value="0.85"/>
+      <param pos="0" name="os.vendor" value="Apple"/>
+      <param pos="0" name="os.family" value="Mac OS X"/>
+      <param pos="0" name="os.product" value="Mac OS X"/>
+      <param pos="0" name="os.version" value="10.2"/>
+   </fingerprint>
+
+   <fingerprint pattern="^osxvers=5">
+      <description>Mac OS X 10.1 (Puma)</description>
+      <example>osxvers=5</example>
+      <param pos="0" name="os.certainty" value="0.85"/>
+      <param pos="0" name="os.vendor" value="Apple"/>
+      <param pos="0" name="os.family" value="Mac OS X"/>
+      <param pos="0" name="os.product" value="Mac OS X"/>
+      <param pos="0" name="os.version" value="10.1"/>
+   </fingerprint>
+
+   <fingerprint pattern="^osxvers=4">
+      <description>Mac OS X 10.0 (Cheetah)</description>
+      <example>osxvers=4</example>
+      <param pos="0" name="os.certainty" value="0.85"/>
+      <param pos="0" name="os.vendor" value="Apple"/>
+      <param pos="0" name="os.family" value="Mac OS X"/>
+      <param pos="0" name="os.product" value="Mac OS X"/>
+      <param pos="0" name="os.version" value="10.0"/>
+   </fingerprint>
+
+
+   <fingerprint pattern="^model=AirPort.*$">
+      <description>AirPort</description>
+      <!-- TODO: example -->
+      <param pos="0" name="os.certainty" value="0.8"/>
```

Comment:
Why 0.8?  For that matter, why specify a certainty for any of these, especially if for the {{_workstation_txt}} records you aren't specifying a certainty at all.